From 879b4673d06ad8e73edf5c069ec6d165bea9fa3e Mon Sep 17 00:00:00 2001
From: Arun P Madhavan <arun_madhavan@comcast.com>
Date: Wed, 21 Sep 2022 20:29:24 +0000
Subject: [PATCH] Fix for lambda crash error RDKTV-16258
Reason for change: Rebased to match the Thunder GITHASH.
Initial commit by muhammed rafi c <muhammed_rafi@comcast.com>
Test Procedure: Please see ticket for details.
Risks: Medium
Source: Comcast
Upstream-Status: Pending
Priority: P1
Signed-off-by: Arun Madhavan <arun_madhavan@comcast.com>
---
 Tools/ProxyStubGenerator/StubGenerator.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)
diff --git a/Tools/ProxyStubGenerator/StubGenerator.py b/Tools/ProxyStubGenerator/StubGenerator.py
index 66f10227..26d13a52 100755
--- a/Tools/ProxyStubGenerator/StubGenerator.py
+++ b/Tools/ProxyStubGenerator/StubGenerator.py
@@ -924,7 +924,11 @@ def GenerateStubs(output_file, source_file, includePaths = [], defaults="", scan
                                     p.name)
                                 emit.IndentDec()
                                 emit.Line("}")
-
+                                emit.Line("else {")
+                                emit.IndentInc()
+                                emit.Line("TRACE_L1(\"Failed to get instance of %s proxy\");" % iface_name)
+                                emit.IndentDec()
+                                emit.Line("}")
                 else:
                     log.Print("stubbed method %s" % m.full_name, source_file)
                     emit.Line("// RPC::Data::Input& input(message->Parameters());")
-- 
2.17.1

